crates/autopilot/src/database/competition.rs

```diff
@@ -117,6 +127,17 @@ impl super::Postgres {
         .await
         .context("auction_prices::insert")?;
 
+        database::settlement_call_data::insert(
+            &mut ex,
+            SettlementCallData {
+                auction_id: competition.auction_id,
+                call_data: competition.call_data.clone(),
+                uninternalized_call_data: competition.uninternalized_call_data.clone(),
+            },
+        )
+        .await
+        .context("settlement_call_data::insert")?;
```

Comment:
Could you add another verification step [here](https://github.com/cowprotocol/services/blob/main/crates/e2e/tests/e2e/colocation_univ2.rs#L101-L122) to e2e test this?